05_stringer/marksomething/main_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStringer(t *testing.T) {
+	testCases := map[string]struct {
+		arg      IPAddr
+		expected string
+	}{
+		"Empty": {
+			arg:      IPAddr{},
+			expected: "0.0.0.0",
+		},
+		"Partial": {
+			arg:      IPAddr{127},
+			expected: "127.0.0.0",
+		},
+		"LocalHost": {
+			arg:      IPAddr{127, 0, 0, 1},
+			expected: "127.0.0.1",
+		},
+		"Common Local": {
+			arg:      IPAddr{192, 168, 0, 1},
+			expected: "192.168.0.1",
+		},
+		"Common Local Broadcast": {
+			arg:      IPAddr{192, 168, 0, 255},
+			expected: "192.168.0.255",
+		},
+	}
+
+	for testName, tC := range testCases {
+		testCase := tC
+		t.Run(testName, func(t *testing.T) {
+			actualFmt := fmt.Sprint(testCase.arg)
+			actualString := testCase.arg.String()
+
+			expected := testCase.expected
+			assert.Equal(t, expected, actualFmt)
+			assert.Equal(t, expected, actualString)
+
```

Comment:
Little nit, but can you please remove the extra line.

Reply:
@rokane Have removed extra line, can you please reapprove.